views/result.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Let's find out who won!!!!</h1>
```

Comment:
If statements are such a good idea  to include in the erb file.

Also, the erb files are named really clearly